config/routes.rb

```diff
@@ -11,9 +11,9 @@
   namespace :admin do
     get '/', to: 'dashboard#index'
     resources :users, only: [:index, :show]
-    get '/merchants', to: 'merchants#index'
-    get '/merchants/:id', to: 'merchants#show'
-    patch '/merchants/:id', to: 'merchants#update'
+    resources :merchants, only: [:index, :show, :update] do
+      resources :items
```

Comment:
This allows admins to interact with items with the same CRUD functionality as a mercahnt.